db/public.sql

```diff
@@ -32,6 +32,12 @@ CREATE TABLE IF NOT EXISTS public.data (
     timeseries INT4 NOT NULL,
     obstime TIMESTAMPTZ NOT NULL,
     obsvalue REAL,
+    -- This value should be treated as an absolute assertion of the data's quality but rather our
```

Comment:
Guess it should say: "... should _not_ be treated ..."

Reply:
Good catch, fixed!